src/app/screens/LNURLAuth/index.tsx

```diff
@@ -88,13 +89,18 @@ function LNURLAuth() {
         <>
           <Container isScreenView maxWidth="sm">
             <div>
-              <PublisherCard
-                title={origin.name}
-                image={origin.icon}
-                url={details.domain}
-              />
+              {origin && (
+                <PublisherCard
+                  title={origin.name}
+                  image={origin.icon}
+                  url={details.domain}
+                />
+              )}
+
               <ContentMessage
-                heading={`${origin.name} asks you to login to`}
+                heading={`${
+                  origin ? origin.name : details.domain
```

Comment:
maybe we rephrase the text here because now we might show twice the domain. 
maybe something like: Do you want to login to: ...?

Reply:
Yes, it would. Updated it.